mesonbuild/backend/ninjabackend.py

```diff
@@ -44,11 +43,14 @@
 FORTRAN_USE_PAT = r"^\s*use,?\s*(?:non_intrinsic)?\s*(?:::)?\s*(\w+)"
 
 if mesonlib.is_windows():
+    # FIXME: can't use quote_arg on Windows just yet; there are a number of existing workarounds
+    # throughout the codebase that cumulatively make the current code work (see, e.g. Backend.escape_extra_args
+    # and NinjaBuildElement.write below) and need to be properly untangled before attempting this
```

Comment:
I did some (most?) of that untangling as part of #5245, maybe that's something I should look at extracting from that PR.